### PR TITLE
chore(actions): move bdd tests to github actions

### PR DESCRIPTION
**Pull request by shubham14bajpai**
Signed-off-by: shubham <shubham.bajpai@mayadata.io>

**What this PR does?**:
This PR moves the bdd tests to github actions.

**Does this PR require any upgrade changes?**:

**If the changes in this PR are manually verified, list down the scenarios covered:**:

**Any additional information for your reviewer?** :
_Mention if this PR is part of any design or a continuation of previous PRs_

**Checklist:**
- [ ] Fixes #<issue number>
- [ ] PR Title follows the convention of  `<type>(<scope>): <subject>`
- [ ] Has the change log section been updated?
- [ ] Commit has unit tests
- [ ] Commit has integration tests
- [ ] (Optional) Are upgrade changes included in this PR? If not, mention the issue/PR to track:
- [ ] (Optional) If documentation changes are required, which issue on https://github.com/openebs/openebs-docs is used to track them:

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -44,9 +44,56 @@ jobs:
...
+        with:
+          minikube version: v1.16.0
+          kubernetes version: ${{ matrix.kubernetes }}
+          github token: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Setting environment variables
+        run: |
+          echo "KUBECONFIG=$HOME/.kube/config" >> $GITHUB_ENV
+          echo "OPENEBS_NAMESPACE=openebs" >> $GITHUB_ENV
+          echo "NodeID=$HOSTNAME" >> $GITHUB_ENV
```
Comment (pawanpraka1) [inline .github/workflows/build.yml:81]: NodeId is not needed anymore. Please remove it.

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -19,8 +19,55 @@ on:
       - 'v*'
 
 jobs:
+  bdd-test:
```
Comment (pawanpraka1) [inline .github/workflows/release.yml:22]: do we need to run it at the release time also? we have already run this at the build time?

Comment (shubham14bajpai) [reply .github/workflows/release.yml]: Yes not required

**Review by pawanpraka1 — APPROVED** [review]
looks good.

**Pull request merged**